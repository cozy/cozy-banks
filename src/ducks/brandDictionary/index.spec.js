import { getBrands, matchBrands, findMatchingBrand } from '.'
import brands from './brands'

const getFilteredBrands = () => getBrands(brand => brand.name === 'Filtered')

describe('brandDictionary', () => {
  describe('getBrands', () => {
    it('Should return brands', () => {
      expect(getBrands()).toBe(brands)
    })
    it('Should return brands filtered', () => {
      expect(getFilteredBrands()).toEqual([])
    })
  })

  describe('matchBrands', () => {
    it('Should return true if label match with regex', () => {
      expect(matchBrands(getBrands(), 'Cpam Des Yvelines')).toBeTruthy()
      expect(matchBrands(getBrands(), 'C.P.A.M Des Yvelines')).toBeTruthy()
      expect(matchBrands(getBrands(), 'C.P.AaM Des Yvelines')).toBeFalsy()
      expect(
        matchBrands(getBrands(), 'caisse primaire Des Yvelines')
      ).toBeTruthy()
      expect(matchBrands(getBrands(), 'Gandi Paris')).toBeFalsy()
    })

    it('Should return false if brand is filtered', () => {
      expect(matchBrands(getFilteredBrands(), 'Cpam Des Yvelines')).toBeFalsy()
    })
  })

  describe('findMatchingBrand', () => {
    it('Should return brand information if label match with regex', () => {
      expect(findMatchingBrand(getBrands(), 'Gandi Paris')).toBeFalsy()
      expect(
        findMatchingBrand(getBrands(), 'Cpam Des Yvelines').konnectorSlug
      ).toBe('ameli')
      expect(findMatchingBrand(getBrands(), 'Free Telecom').konnectorSlug).toBe(
        'free'
      )
      expect(
        findMatchingBrand(getBrands(), 'Free HautDebit').konnectorSlug
      ).toBe('free')
      expect(findMatchingBrand(getBrands(), 'Free Mobile').konnectorSlug).toBe(
        'freemobile'
      )
    })
  })
})
