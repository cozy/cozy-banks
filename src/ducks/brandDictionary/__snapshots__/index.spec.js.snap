// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`brandDictionary getBrandsWithInstallationInfo Should return the list of brands with a isInstalled boolean property 1`] = `
Array [
  Object {
    "isInstalled": false,
    "konnectorSlug": "amazon",
    "name": "Amazon",
    "regexp": "\\\\bamazo?n\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": true,
    "konnectorSlug": "ameli",
    "name": "Ameli",
    "regexp": "(cpam|c\\\\.p\\\\.a\\\\.m|caisse primaire)",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "aprr",
    "name": "APRR",
    "regexp": "\\\\baprr\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "ascoresante",
    "name": "Ascore Santé Prévoyance",
    "regexp": "\\\\baxiome\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "auchandrive",
    "name": "Auchan Drive",
    "regexp": "\\\\bauchan\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "autolib",
    "maintenance": true,
    "name": "Autolib",
    "regexp": "\\\\bautolib\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bipandgo",
    "name": "Bipandgo",
    "regexp": "\\\\bbipandgo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bookingdotcom",
    "maintenance": true,
    "name": "Booking.com",
    "regexp": "\\\\bbooking.com\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "boulanger",
    "name": "Boulanger",
    "regexp": "\\\\bboulanger\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bouyguesbox",
    "name": "Bouygues Box",
    "regexp": "\\\\bbouyg",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bouyguestelecom",
    "name": "Bouygues Telecom",
    "regexp": "\\\\bbouyg",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "caf",
    "name": "CAF",
    "regexp": "\\\\bcaf\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "cdiscount",
    "name": "Cdiscount",
    "regexp": "\\\\bcdiscount\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "cesu",
    "name": "CESU",
    "regexp": "\\\\bcncesu\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "cultura",
    "maintenance": true,
    "name": "Cultura",
    "regexp": "\\\\bcultura\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "darty",
    "maintenance": true,
    "name": "Darty",
    "regexp": "\\\\bdarty\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "deezer",
    "maintenance": true,
    "name": "Deezer",
    "regexp": "\\\\bdeezer\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "deliveroo",
    "maintenance": true,
    "name": "Deliveroo",
    "regexp": "\\\\bdeliveroo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "directenergie",
    "name": "Direct Energie",
    "regexp": "\\\\bdirect energie\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "edf",
    "name": "EDF",
    "regexp": "\\\\bedf\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "ekwateur",
    "name": "Ekwateur",
    "regexp": "\\\\bekwateur\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "electricitestrasbourg",
    "maintenance": true,
    "name": "Electricité de Strabourg",
    "regexp": "es energies strasbourg\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "enercoop",
    "name": "Enercoop",
    "regexp": "\\\\benercoop\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "engie",
    "maintenance": true,
    "name": "Engie",
    "regexp": "\\\\bengie\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "ensap",
    "name": "Ensap",
    "regexp": "\\\\b(ddfip|drfip)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "eovimcdparticuliers",
    "name": "Eovi MCD Particuliers",
    "regexp": "\\\\beovi mcd mutuelle\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "free",
    "name": "Free",
    "regexp": "\\\\bfree (telecom|hautdebit)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "freemobile",
    "name": "Free Mobile",
    "regexp": "\\\\bfree mobile\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "generali",
    "name": "Generali",
    "regexp": "\\\\bgenerali\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "harmonie",
    "name": "Harmonie Mutuelle",
    "regexp": "\\\\bharmonie\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "\\\\bhumanis\\\\b",
    "name": "Humanis",
    "regexp": "\\\\bhumanis\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "inulogic",
    "name": "Inulogic",
    "regexp": "\\\\binulogic\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "impots",
    "name": "Impôts",
    "regexp": "(impot|impots|dgfip|d\\\\.g\\\\.f\\\\.i\\\\.p|ddfip|d\\\\.d\\\\.f\\\\.i\\\\.p|drfip|d\\\\.r\\\\.f\\\\.i\\\\.p|tresor public|finances pub|finances publiques)",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "justeat",
    "name": "JustEat",
    "regexp": "\\\\bjust eat\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "lamutuellegenerale",
    "name": "La mutuelle générale",
    "regexp": "\\\\bla mutuelle gen",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "ldlc",
    "maintenance": true,
    "name": "LDLC",
    "regexp": "\\\\bldlc\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "mondediplo",
    "name": "Le Monde Diplomatique",
    "regexp": "\\\\bmonde-diplomatique\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "leclercdrive",
    "maintenance": true,
    "name": "Leclerc Drive",
    "regexp": "\\\\bleclerc\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "malentille",
    "name": "Ma lentille",
    "regexp": "\\\\bdigital eyewear\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "maifecheancier",
    "name": "Maif - Echeancier",
    "regexp": "\\\\bmaif\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "macif",
    "maintenance": true,
    "name": "Macif",
    "regexp": "\\\\bmacif\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "malakoffmederic",
    "name": "Malakoff Mederic",
    "regexp": "\\\\bmalakoff\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "msf",
    "name": "Médecins sans frontières",
    "regexp": "\\\\b(medecins sans frontieres|msf)\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "mediapart",
    "name": "Mediapart",
    "regexp": "\\\\bmediapart\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "mgen",
    "name": "MGEN",
    "regexp": "\\\\bmgen\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "misterauto",
    "name": "Mister Auto",
    "regexp": "\\\\bmister auto\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "netflix",
    "name": "Netflix",
    "regexp": "\\\\bnetflix\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "numericable",
    "name": "Numéricable",
    "regexp": "\\\\bnumericable\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "o2",
    "name": "O2",
    "regexp": "^o2\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "opngo",
    "name": "Opngo",
    "regexp": "\\\\bopngo\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "ovh",
    "name": "OVH",
    "regexp": "\\\\bovh\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "planeteoui",
    "name": "Planete OUI",
    "regexp": "\\\\boui energy\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "probikeshop",
    "name": "Probikeshop",
    "regexp": "\\\\bprobikeshop\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "productboard",
    "name": "ProductBoard",
    "regexp": "\\\\bproductboard\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "saur",
    "name": "SAUR",
    "regexp": "\\\\bsaur\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "scaleway",
    "name": "Scaleway",
    "regexp": "\\\\bscaleway\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "sfr",
    "name": "SFR",
    "regexp": "\\\\bsfr\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "showroomprive",
    "name": "Showroom Privé",
    "regexp": "\\\\bshowroomprive\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "sncf",
    "name": "SNCF",
    "regexp": "\\\\bsncf\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "spartoo",
    "name": "Spartoo",
    "regexp": "\\\\bspartoo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "stickersdiscount",
    "name": "Stickers Discount",
    "regexp": "\\\\bakoufen angers\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "tagandpass",
    "name": "Tag&Pass",
    "regexp": "\\\\bsemitag\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "theoldreader",
    "name": "The Old Reader",
    "regexp": "\\\\btheoldreader\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "thomann",
    "name": "Thomann",
    "regexp": "\\\\bthomann\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "trainline",
    "name": "Trainline",
    "regexp": "\\\\btrainline\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "unibet",
    "name": "Unibet",
    "regexp": "\\\\bunibet\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "venteprivee",
    "name": "Veepee",
    "regexp": "\\\\b(Vente\\\\-privee\\\\.com|veepee)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "veoliaeau",
    "name": "Veolia Eau",
    "regexp": "\\\\bveolia\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "vinciautoroute",
    "name": "Vinci autoroute télépéage",
    "regexp": "\\\\b(vinci|escota)\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "zalando",
    "maintenance": true,
    "name": "Zalando",
    "regexp": "\\\\bzalando\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "zooplus",
    "name": "ZooPlus",
    "regexp": "\\\\bzooplus\\\\b",
  },
]
`;

exports[`brandDictionary getInstalledBrands Should return the installed brands 1`] = `
Array [
  Object {
    "health": true,
    "isInstalled": true,
    "konnectorSlug": "ameli",
    "name": "Ameli",
    "regexp": "(cpam|c\\\\.p\\\\.a\\\\.m|caisse primaire)",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "boulanger",
    "name": "Boulanger",
    "regexp": "\\\\bboulanger\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "mediapart",
    "name": "Mediapart",
    "regexp": "\\\\bmediapart\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "ovh",
    "name": "OVH",
    "regexp": "\\\\bovh\\\\b",
  },
  Object {
    "isInstalled": true,
    "konnectorSlug": "zalando",
    "maintenance": true,
    "name": "Zalando",
    "regexp": "\\\\bzalando\\\\b",
  },
]
`;

exports[`brandDictionary getInstalledBrands Should return the not installed brands 1`] = `
Array [
  Object {
    "isInstalled": false,
    "konnectorSlug": "amazon",
    "name": "Amazon",
    "regexp": "\\\\bamazo?n\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "aprr",
    "name": "APRR",
    "regexp": "\\\\baprr\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "ascoresante",
    "name": "Ascore Santé Prévoyance",
    "regexp": "\\\\baxiome\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "auchandrive",
    "name": "Auchan Drive",
    "regexp": "\\\\bauchan\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "autolib",
    "maintenance": true,
    "name": "Autolib",
    "regexp": "\\\\bautolib\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bipandgo",
    "name": "Bipandgo",
    "regexp": "\\\\bbipandgo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bookingdotcom",
    "maintenance": true,
    "name": "Booking.com",
    "regexp": "\\\\bbooking.com\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bouyguesbox",
    "name": "Bouygues Box",
    "regexp": "\\\\bbouyg",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "bouyguestelecom",
    "name": "Bouygues Telecom",
    "regexp": "\\\\bbouyg",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "caf",
    "name": "CAF",
    "regexp": "\\\\bcaf\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "cdiscount",
    "name": "Cdiscount",
    "regexp": "\\\\bcdiscount\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "cesu",
    "name": "CESU",
    "regexp": "\\\\bcncesu\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "cultura",
    "maintenance": true,
    "name": "Cultura",
    "regexp": "\\\\bcultura\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "darty",
    "maintenance": true,
    "name": "Darty",
    "regexp": "\\\\bdarty\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "deezer",
    "maintenance": true,
    "name": "Deezer",
    "regexp": "\\\\bdeezer\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "deliveroo",
    "maintenance": true,
    "name": "Deliveroo",
    "regexp": "\\\\bdeliveroo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "directenergie",
    "name": "Direct Energie",
    "regexp": "\\\\bdirect energie\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "edf",
    "name": "EDF",
    "regexp": "\\\\bedf\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "ekwateur",
    "name": "Ekwateur",
    "regexp": "\\\\bekwateur\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "electricitestrasbourg",
    "maintenance": true,
    "name": "Electricité de Strabourg",
    "regexp": "es energies strasbourg\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "enercoop",
    "name": "Enercoop",
    "regexp": "\\\\benercoop\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "engie",
    "maintenance": true,
    "name": "Engie",
    "regexp": "\\\\bengie\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "ensap",
    "name": "Ensap",
    "regexp": "\\\\b(ddfip|drfip)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "eovimcdparticuliers",
    "name": "Eovi MCD Particuliers",
    "regexp": "\\\\beovi mcd mutuelle\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "free",
    "name": "Free",
    "regexp": "\\\\bfree (telecom|hautdebit)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "freemobile",
    "name": "Free Mobile",
    "regexp": "\\\\bfree mobile\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "generali",
    "name": "Generali",
    "regexp": "\\\\bgenerali\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "harmonie",
    "name": "Harmonie Mutuelle",
    "regexp": "\\\\bharmonie\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "\\\\bhumanis\\\\b",
    "name": "Humanis",
    "regexp": "\\\\bhumanis\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "inulogic",
    "name": "Inulogic",
    "regexp": "\\\\binulogic\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "impots",
    "name": "Impôts",
    "regexp": "(impot|impots|dgfip|d\\\\.g\\\\.f\\\\.i\\\\.p|ddfip|d\\\\.d\\\\.f\\\\.i\\\\.p|drfip|d\\\\.r\\\\.f\\\\.i\\\\.p|tresor public|finances pub|finances publiques)",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "justeat",
    "name": "JustEat",
    "regexp": "\\\\bjust eat\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "lamutuellegenerale",
    "name": "La mutuelle générale",
    "regexp": "\\\\bla mutuelle gen",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "ldlc",
    "maintenance": true,
    "name": "LDLC",
    "regexp": "\\\\bldlc\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "mondediplo",
    "name": "Le Monde Diplomatique",
    "regexp": "\\\\bmonde-diplomatique\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "leclercdrive",
    "maintenance": true,
    "name": "Leclerc Drive",
    "regexp": "\\\\bleclerc\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "malentille",
    "name": "Ma lentille",
    "regexp": "\\\\bdigital eyewear\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "maifecheancier",
    "name": "Maif - Echeancier",
    "regexp": "\\\\bmaif\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "macif",
    "maintenance": true,
    "name": "Macif",
    "regexp": "\\\\bmacif\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "malakoffmederic",
    "name": "Malakoff Mederic",
    "regexp": "\\\\bmalakoff\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "msf",
    "name": "Médecins sans frontières",
    "regexp": "\\\\b(medecins sans frontieres|msf)\\\\b",
  },
  Object {
    "health": true,
    "isInstalled": false,
    "konnectorSlug": "mgen",
    "name": "MGEN",
    "regexp": "\\\\bmgen\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "misterauto",
    "name": "Mister Auto",
    "regexp": "\\\\bmister auto\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "netflix",
    "name": "Netflix",
    "regexp": "\\\\bnetflix\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "numericable",
    "name": "Numéricable",
    "regexp": "\\\\bnumericable\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "o2",
    "name": "O2",
    "regexp": "^o2\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "opngo",
    "name": "Opngo",
    "regexp": "\\\\bopngo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "planeteoui",
    "name": "Planete OUI",
    "regexp": "\\\\boui energy\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "probikeshop",
    "name": "Probikeshop",
    "regexp": "\\\\bprobikeshop\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "productboard",
    "name": "ProductBoard",
    "regexp": "\\\\bproductboard\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "saur",
    "name": "SAUR",
    "regexp": "\\\\bsaur\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "scaleway",
    "name": "Scaleway",
    "regexp": "\\\\bscaleway\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "sfr",
    "name": "SFR",
    "regexp": "\\\\bsfr\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "showroomprive",
    "name": "Showroom Privé",
    "regexp": "\\\\bshowroomprive\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "sncf",
    "name": "SNCF",
    "regexp": "\\\\bsncf\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "spartoo",
    "name": "Spartoo",
    "regexp": "\\\\bspartoo\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "stickersdiscount",
    "name": "Stickers Discount",
    "regexp": "\\\\bakoufen angers\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "tagandpass",
    "name": "Tag&Pass",
    "regexp": "\\\\bsemitag\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "theoldreader",
    "name": "The Old Reader",
    "regexp": "\\\\btheoldreader\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "thomann",
    "name": "Thomann",
    "regexp": "\\\\bthomann\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "trainline",
    "name": "Trainline",
    "regexp": "\\\\btrainline\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "unibet",
    "name": "Unibet",
    "regexp": "\\\\bunibet\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "venteprivee",
    "name": "Veepee",
    "regexp": "\\\\b(Vente\\\\-privee\\\\.com|veepee)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "veoliaeau",
    "name": "Veolia Eau",
    "regexp": "\\\\bveolia\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "vinciautoroute",
    "name": "Vinci autoroute télépéage",
    "regexp": "\\\\b(vinci|escota)\\\\b",
  },
  Object {
    "isInstalled": false,
    "konnectorSlug": "zooplus",
    "name": "ZooPlus",
    "regexp": "\\\\bzooplus\\\\b",
  },
]
`;
